src/main/java/address/controller/RootLayoutController.java

```diff
@@ -119,6 +121,8 @@ private void handleSaveAs() {
         }
 
         modelManager.setPrefsSaveLocation(file.getPath());
+        // TODO temp patch to create cloud file if non found
+        raise(new CreateAddressBookOnRemoteRequestEvent(new CompletableFuture<>(), file.getName()));
```

Comment:
Yes, this is a hack to let the user have some way of creating remote files when they never had one.